atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/KvsMigrationCommand.java

```diff
@@ -126,6 +126,8 @@ public int execute(AtlasDbServices fromServices, AtlasDbServices toServices) {
         }
         if (migrate) {
             migrator.migrate();
+
+            migrator.cleanup();
```

Comment:
You can still resume a migration as it uses checkpointing to record it's progress.  So as long as you don't call setup again you should be able to call migrate as many times as you would like (until it completes).